statchatui/src/SignIn/SignIn.jsx

```diff
@@ -18,6 +18,7 @@ const SignIn = () => {
           "Content-Type": "application/json",
         },
         body: JSON.stringify(formData),
+        credentials: "include",
```

Comment:
Maybe you can create a task to move all these `fetch` calls into a helper function?

Reply:
ok